src/style.css

```diff
@@ -1,3 +1,62 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For instance, you could specify 'Roboto', 'Arial', sans-serif.